app/views/layouts/_collection_resources.html.erb

```diff
@@ -7,50 +7,65 @@
     {
       name: "Lettings log for tenants (2023/24)",
       href: download_23_24_lettings_form_path,
-      metadata: "PDF, 278 KB, 8 pages",
+      metadata: metadata_helper("2023_24_lettings_paper_form.pdf", 8)
     },
     {
       name: "Lettings bulk upload template (2023/24) – New question ordering",
       href: download_23_24_lettings_bulk_upload_template_path,
-      metadata: "Microsoft Excel, 15 KB",
+      metadata: metadata_helper("bulk-upload-lettings-template-2023-24.xlsx"),
     },
     {
       name: "Lettings bulk upload template (2023/24)",
       href: download_23_24_lettings_bulk_upload_legacy_template_path,
-      metadata: "Microsoft Excel, 15 KB",
+      metadata: metadata_helper("bulk-upload-lettings-legacy-template-2023-24.xlsx"),
     },
     {
       name: "Lettings bulk upload specification (2023/24)",
       href: download_23_24_lettings_bulk_upload_specification_path,
-      metadata: "Microsoft Excel, 90 KB",
+      metadata: metadata_helper("bulk-upload-lettings-specification-2023-24.xlsx"),
+    },
+  ]) %>
+
+  <h3 class="govuk-heading-s">Sales 2023/24</h3>
+  <%= render DocumentListComponent.new(items: [
+    {
+      name: "Sales log for buyers (2023/24)",
+      href: download_23_24_sales_form_path,
+      metadata: metadata_helper("2023_24_sales_paper_form.pdf", 8),
+    },
+    {
+      name: "Sales bulk upload template (2023/24) – New question ordering",
+      href: download_23_24_sales_bulk_upload_template_path,
+      metadata: metadata_helper("bulk-upload-sales-template-2023-24.xlsx"),
+    },
+    {
+      name: "Sales bulk upload template (2023/24)",
+      href: download_23_24_sales_bulk_upload_legacy_template_path,
+      metadata: metadata_helper("bulk-upload-sales-legacy-template-2023-24.xlsx"),
+    },
+    {
+      name: "Sales bulk upload specification (2023/24)",
+      href: download_23_24_sales_bulk_upload_specification_path,
+      metadata: metadata_helper("bulk-upload-sales-specification-2023-24.xlsx"),
     },
   ]) %>
 
   <h3 class="govuk-heading-s">Lettings 2022/23</h3>
   <%= render DocumentListComponent.new(items: [
     {
       name: "Lettings log for tenants (2022/23)",
-      href: "https://core.communities.gov.uk/public/download/guides-and-manuals/2022-23%20Lettings%20paper%20form.pdf?download-format=pdf",
```

Comment:
I think this is fine, but happy to have a second opinion in case there's a reason for this being pulled from core that I've missed